src/api/json/catalog.json

```diff
@@ -1069,7 +1069,11 @@
       "fileMatch": [
         "feed.json"
       ],
-      "url": "https://json.schemastore.org/feed"
+      "url": "https://json.schemastore.org/feed",
+      "versions": {
+        "1": "http://json.schemastore.org/feed-1",
```

Comment:
Isnt' this the same as `https://json.schemastore.org/feed`? Why not use the same URL?

Reply:
No. I made `/feed` be the latest version, and users can specify a more specific version by adding an appropriate suffix. The content of `/feed` is the v1.1 spec.

Reply:
Would it be more reasonable to just delete all the original v1 content and only have the v1.1 schema?

Reply:
It's up to you, but our recommendation is to leave the "url" value the latest schema and then use the "versions" array to list all schemas. That way you'd have **/feed** and **/feed-1.0** only and the content of **/feed** would be version 1.1

Reply:
So would I still have a `1.1` key in the `versions` listing? How would a user indicate that they want to conform to v1.1 and not a later version?

Reply:
Yes, still keep 1.1 in the versions list